dev_suites/smart/app_launch_test.rb

```diff
@@ -0,0 +1,23 @@
+module SMART
+  class AppLaunchTest < Inferno::Test
+    title 'EHR server redirects client browser to Inferno app launch URI'
+    description %(
+      Client browser sent from EHR server to app launch URI of client app as
+      described in SMART EHR Launch Sequence.
+    )
+    id :smart_app_launch
+
+    input :url
+    receives_request :launch
+
+    run do
+      wait(
+        identifier: url,
```

Comment:
I think this is also an issue in current inferno, but it seems like you can't have two instances of inferno testing smart launches on the same server at the same time. Is that something we care about?